### PR TITLE
Added routing helpers to actions

### PR DESCRIPTION
**Pull request by AlfonsoUceda**
With this PR you can use routing helpers inside actions.

```ruby
module Collaboration::Controllers::RedirectedRoutes
  class Index
    include Collaboration::Action

    def call(params)
      redirect_to routes.legacy_path
    end
  end
end
```

### CODE REVIEW

**Comment by jodosha** [comment]
@AlfonsoUceda Please grep the codebase and update fixtures [like this](https://github.com/lotus/lotus/blob/master/test/fixtures/collaboration/apps/web/app/controllers/books.rb#L58):

```ruby
# from
redirect_to Collaboration::Routes.url(:book, id: @book.id)

# to
redirect_to routes.book_url(id: @book.id)
```

**Comment by AlfonsoUceda** [comment]
@jodosha done!

**Comment by joneslee85** [comment]
@AlfonsoUceda :+1:

File: test/integration/full_stack_test.rb [hunk 1]
```diff
@@ -233,5 +233,11 @@ def response
     end
   end
 
+  it 'test route helpers in actions' do
+    get '/redirected_routes'
+
+    response.status.must_equal 302
+    response.headers["Location"].must_equal "/action_legacy"
```
Comment (joneslee85) [inline test/integration/full_stack_test.rb]: _ntipick_ why not go with single quote?

File: lib/lotus/loader.rb [hunk 2]
```diff
@@ -60,6 +61,7 @@ def _configure_controller_framework!
             cookies config.cookies.default_options
           end
           prepare { include Lotus::Action::Session } if config.sessions.enabled?
+          prepare { include Lotus::Action::RoutingHelpers }
```
Comment (joneslee85) [inline lib/lotus/loader.rb]: @AlfonsoUceda what do you think about including this helper in view framework too?

Comment (AlfonsoUceda) [inline lib/lotus/loader.rb]: @joneslee85 in this line, routing helpers are loaded for views https://github.com/lotus/lotus/blob/master/lib/lotus/generators/slice/application.rb.tt#L209

**Comment by jodosha** [comment]
@AlfonsoUceda We're doing great! Lotus is getting better and better. Thank you :clap:

**Pull request merged**